src/onepay-checkout.js

```diff
@@ -919,4 +891,31 @@ function connectSocket(onepay) {
   client.connect();
 }
 
+let CheckoutModal = zoid.create({
```

Comment:
Why do you repeat the `zoid.create` stuff on both files (merchant js and iframe js)? 

Isn't easier for future maintenance to create a separate file which exports the `CheckoutModal` and then simply do `let CheckoutModal = require('checkout-modal-component')` on both files?

Reply:
I have extract to separated file now